contracts/test/mocks/AdjustableRoundsManager.sol

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.8.9;
 
-import "./RoundsManager.sol";
+import "../../rounds/RoundsManager.sol";
 
 contract AdjustableRoundsManager is RoundsManager {
```

Comment:
Don't we need to update some paths also in some `js/ts` files? I see it's used in a number of files, e.g., [here](https://github.com/livepeer/protocol/blob/c3b6273651801121c0fa533d4f68dd1642d16a2e/test/gas-report/redeemTicket.js#L53).

Reply:
It's not necessary because those hardhat/ethers methods are dependent on the name of the compiled contract rather than its path